app/views/tags/index.html.erb

```diff
@@ -20,8 +20,9 @@
 <% end %>
 
 <main id="main-content" class="crayons-layout crayons-layout--1-col">
-  <header class="px-2 m:px-0">
+  <header class="px-2 m:px-0 flex items-center justify-between">
     <h1>Top tags</h1>
+    <%= link_to "Following tags", dashboard_following_tags_path, class: "crayons-btn crayons-btn--ghost" %>
```

Comment:
nitpick (suggestion): The wording is not clear. Consider _Followed Tags_ or _Tags I Follow_.

Reply:
this is the label we use in Dashboard so if we change it in one place we should do it in others too.. as i'm not an expert when it comes to proper English I'd rely on your suggestion but would love to double check if you think it also makes sense to update it in Dashboard.

Reply:
I think it makes sense to change it there too. Followed Tags probably makes the most sense and it's fewer characters. Also good catch on keeping naming consistent!

Reply:
I'm looking at Twitter, Instagram and GitHub now and they all use "Following" in this scenario.. And I'm now officially confused :D